WalletWasabi/Legal/Assets/LegalDocumentsWw2.txt

```diff
@@ -1,75 +1,279 @@
+
+2 THE COINJOIN COORDINATOR SERVICES
+===================================
+
+2.1 DESCRIPTION OF SERVICES
+    This Software consists of the Client Application and the CoinJoin Coordinator (both as defined below) and functions as a free, open-source, non-custodial desktop Bitcoin wallet. The Software does not constitute an account where we or other third parties serve as financial intermediaries or custodians of Your bitcoins. The Software and our Services are therefore not a bank, a custodian, an exchange, a financial intermediary or a regulated financial institution.
+
+    The client application (“Client Application”) is software, with the sole purpose of allowing You access to the Bitcoin network and our Services (as defined below), without the need or requirement to create or maintain a user account. The application itself is an ordinary Bitcoin desktop wallet with ordinary features. Wasabi Wallet does not store, send or receive bitcoins. This is because bitcoins exist only by virtue of the ownership record maintained in the Bitcoin network. Any transfer of title in bitcoins occurs within the decentralized Bitcoin network, and not in the Wasabi Wallet.
+    The CoinJoin Coordinator Service is an online service that implements trustless CoinJoin to prevent third parties from spying on the Blockchain. Throughout the process, the Service does not initiate or process any standalone transactions whatsoever towards third parties (i.e. non-users of the Service) and therefore does not store or transmit value belonging to others. For further information on how it works please visit our GitHub page: https://github.com/zkSNACKs. You acknowledge that the CoinJoin Service runs in the background by default, while using the Client Application. To avoid fees, you may turn automatic CoinJoin Services off in the settings of the Client Application.
+	The UTXO is an unspent transaction output, the technical term for the amount of digital currency that remains after a cryptocurrency transaction. 
+	The One Hop transaction is when there is one non coinjoined transaction in between two coinjoined transactions.
+
+2.2 FEES
+    Subject to the other provisions of these Terms, including, but not limited to Section 3 on Prohibited Activities, You may freely download and use the Client Application without any charge or fee imposed on You by the Service Provider.
+    The Service Provider does not charge You transaction fees for normal Bitcoin transactions, however You are still subject to transaction fees, charged by the Bitcoin network.
+    Both the Service Provider and the Bitcoin network charge You transaction fees for CoinJoin Bitcoin transactions.
+
+    The CoinJoin fee, charged by the Service Provider is 0.3% per new UTXO, except if:
+     -The transaction comes directly from a previous Wasabi Wallet 1.x CoinJoin output..
+     -The transaction defines as a One Hop transaction.
```

Comment:
I guess here we have to specify that we are talking about Wasabi coinjoin transactions, otherwise one can use any other service to coinjoin and then tries to remix with Wasabi.

Also we didn't mention the free remixes from a WW2 coinjoin transaction.